simulator/fetch/fetch.cpp

```diff
@@ -11,6 +11,11 @@ namespace config {
     static Value<std::string> bp_mode = { "bp-mode", "dynamic_two_bit", "branch prediction mode"};
     static Value<uint32> bp_size = { "bp-size", 128, "BTB size in entries"};
     static Value<uint32> bp_ways = { "bp-ways", 16, "number of ways in BTB"};
+
+    /* Cache parameters */
+    static Value<uint32> size_in_bytes = {"size", 2048, "cache size in bytes"};
```

Comment:
These strings will be command line arguments, so they should be much more informative (see BP example).
You need to specify it is related to instruction cache at least.